WorkingWithAzureDB/WorkingWithAzureDB/Customer.cs

```diff
@@ -11,16 +11,18 @@ namespace WorkingWithAzureDB
 {
     using System;
     using System.Collections.Generic;
-    
+    // Define the Customer type for the DB 
     public partial class Customer
     {
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
+        // COnstructor of the type
         public Customer()
         {
             this.CustomerAddresses = new HashSet<CustomerAddress>();
             this.SalesOrderHeaders = new HashSet<SalesOrderHeader>();
         }
-    
+        
+        // Methods to get information about the customer
```

Comment:
you can use region blocks. see [here](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-region)